run-tests.sh

```diff
@@ -55,6 +59,7 @@ do
     case $arg in
         --check-shellscript) check_script;;
         --check-sphinx) check_sphinx;;
+        --check-lint) check_lint;;
```

Comment:
We should also add `check_lint` to `check_all()` and probably add a new GA CI job as well?

Minor: year was not updated in this file

Reply:
What about GA CI job for linting? Don't we need it?